services/openvpn/service/manager.go

```diff
@@ -91,23 +97,24 @@ func (m *Manager) Serve(providerID identity.Identity) (err error) {
 		return errors.Wrap(err, "failed to add NAT forwarding rule")
 	}
 
-	m.releasePorts = m.mapPort()
+	m.port = m.ports.Acquire()
```

Comment:
afaik the serve is blocking. in this case, we can just do a 
```golang
port := m.ports.Acquire()
defer port.Release()
```
same can be done with the firewall and we then don't have to extend the struct with the redundant field.

Reply:
Although `Release` functionality in the latest version has been removed, it is still needed on the manager instance, because of `Stop`:

`firewall.RemoveInboundRule(m.serviceOptions.Protocol, m.port.Num())`

Reply:
Removed functionality from `Stop()`, replaced with defers.